src/programs/BuddyProject.ts

```diff
@@ -7,21 +7,27 @@ import {
 import { Repository } from "typeorm";
 
 const updateDatabaseWithQuery = (
-  BE: Repository<BuddyProjectEntry>,
+  BuddyEntryRepo: Repository<BuddyProjectEntry>,
   memberId: string,
   buddyId: string,
   BuddyEntry: BuddyProjectEntry
 ) => {
-  BE.createQueryBuilder()
+  BuddyEntryRepo.createQueryBuilder()
     .update(BuddyEntry)
```

Comment:
I am not really familiar with TypeORM but I think the BuddyEntry isn't needed here. In fact I suspect this to be the reason that people aren't actually being matched (however I would like @jamie-legg or @sklirg to double check this):

![grafik](https://user-images.githubusercontent.com/26303198/82036323-0ec64f80-96a1-11ea-88e7-0f484d0b4d6d.png)


```suggestion
    .update()
```